client/dist/index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <head>
   <title>unZwilling Questions</title>
-  <!-- <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet"> -->
-  <!-- <link rel="stylesheet" href="https://use.typekit.net/usx2dju.css"> -->
+  <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
```

Comment:
Nice. I wonder how your font looks like!

Reply:
I don't really like them as much as the built-in futura.